### PR TITLE
Fix colours in the Django admin after 3.2 update

### PR DESCRIPTION
**Pull request by se-bastiaan**
### Summary
Django introduced theming options in version 3.2, however they use some of the same variables 

### How to test
Steps to test the changes you made:
1. Go to the events and pizza admins
2. Check that the theming is correct

### CODE REVIEW

**Comment by JobDoesburg** [comment]
Closes #744 🤯

**Comment by se-bastiaan** [comment]
> Closes #744 🤯

No not yet, this just fixes some problematic overrides

**Comment by ColonelPhantom** [comment]
Looks fine to me, the changes all make sense. With one exception: is manually including the stylesheet in the page necessary? When looking at the code in my browser, I can see it being included twice.

But the changes to the HTML itself and the CSS don't raise any red flags, and testing it seems to work OK.

File: website/pizzas/templates/pizzas/admin/orders.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 {% block extrastyle %}
     {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="/static/admin/css/changelists.css">
```
Comment (ColonelPhantom) [inline website/pizzas/templates/pizzas/admin/orders.html:19]: This extra `link` seems superfluous, since currently the stylesheet gets included twice.

**Review by ColonelPhantom — CHANGES_REQUESTED** [review]
Looks good apart from this!

File: website/pizzas/templates/pizzas/admin/orders.html (hunk 1 above)
Comment (se-bastiaan) [reply website/pizzas/templates/pizzas/admin/orders.html:19]: hmmm weird, I do not understand where the stylesheet comes from. But good catch!

Comment (se-bastiaan) [reply website/pizzas/templates/pizzas/admin/orders.html]: fixed it!

**Review by JobDoesburg — APPROVED** [review]

**Review by ColonelPhantom — APPROVED** [review]
Thing I mentioned was fixed, so looks good!

**Pull request merged**